src/components/User/SignUpForm.js

```diff
@@ -86,6 +86,21 @@ const SignUpForm = ({ onBackClick, user }) => {
 
     setSignUpFormUserData(userData);
 
+    const replicateCandidateData = (user) => {
```

Comment:
It would be nice to ensure this new behavior by covering it with a new test at [App.test.js](https://github.com/Minhacps/votacidade-app/blob/develop/src/App.test.js)

Reply:
Tem razão, adicionei os testes nos últimos commits.
Precisei mudar mais algumas coisas para fazer os testes funcionarem mas agora estamos cobrindo cadastro de eleitoress e candidatos separadamente.